library/public/engine.h

```diff
@@ -120,6 +124,7 @@ class F3D_EXPORT engine
     std::string Description;
     std::vector<std::string> Extensions;
     std::vector<std::string> MimeTypes;
+    std::string PluginName;
```

Comment:
PLuginNames ?

Reply:
No, a plugin has several readers, but a reader is in one single plugin.

Reply:
Right